benchmarks/CMakeLists.txt

```diff
@@ -5,17 +5,17 @@ add_headers (HDR_FILES)
 macro (do_benchmark source)
 	include_directories ("${CMAKE_CURRENT_SOURCE_DIR}")
 	set (SOURCES ${HDR_FILES} benchmarks.c benchmarks.h ${source}.c)
-	add_executable (${source} ${SOURCES})
-	add_dependencies (${source} kdberrors_generated)
+	add_executable (benchmark_${source} ${SOURCES})
+	add_dependencies (benchmark_${source} kdberrors_generated)
 
-	target_link_elektra(${source} elektra-kdb elektra-meta)
+	target_link_elektra(benchmark_${source} elektra-kdb elektra-meta)
 
-	set_target_properties (${source} PROPERTIES
+	set_target_properties (benchmark_${source} PROPERTIES
 			COMPILE_DEFINITIONS HAVE_KDBCONFIG_H)
 
-	get_target_property (command ${source} LOCATION)
+	get_target_property (command benchmark_${source} LOCATION)
 
-	add_custom_target (benchmark_${source}
+	add_custom_target (benchmark_${source}_callgrind
```

Comment:
If I would not rename the custom target it would have the same name as the executable and would lead to a cmake error.